mappings/net/minecraft/item/Item.mapping

```diff
@@ -83,6 +84,7 @@ CLASS none/aeo net/minecraft/item/Item
 		ARG 0 maxStackSize
 	METHOD d getItemById (Ljava/lang/String;)Lnone/aeo;
 		ARG 0 id
+	METHOD e setMaxDamage (I)Lnone/aeo;
```

Comment:
I'd personally like durability the most, though an item's "damage" is used for more than just that, right? (I suppose :+1: this comment if you want `durability` and :-1: if `maxDamage`.)